bin/scripts/gotta-patch-em-all-font-patcher!.sh

```diff
@@ -157,7 +159,7 @@ function generate_info {
   # if first time with this font then re-build parent dir readme, else skip:
   if [[ $config_parent_dir != "$last_parent_dir"  && (! $is_unpatched_fonts_root) ]];
   then
-    echo "Re-generate parent directory readme"
+    echo "[Nerd Fonts] Re-generate parent directory readme"
```

Comment:
This line did not start with the pattern ("#" ) so instead of modifying the expected string I hard coded it. I am happy to make the replacement, however.